src/style-spec/reference/v8.json

```diff
@@ -1006,7 +1006,7 @@
     "symbol-avoid-edges": {
       "type": "boolean",
       "default": false,
-      "doc": "If true, the symbols will not cross tile edges to avoid mutual collisions. Recommended in layers that don't have enough padding in the vector tile to prevent collisions, or if it is a point symbol layer placed after a line symbol layer.",
+      "doc": "If true, the symbols will not cross tile edges to avoid mutual collisions. Recommended in layers that don't have enough padding in the vector tile to prevent collisions, or if it is a point symbol layer placed after a line symbol layer. When using a client that supports global collision detection, like Mapbox GL JS version 0.42.0 or greater, enabling this property is not needed to prevent clipped labels at tile boundaries.",
```

Comment:
nit: Instead of “like” I would suggest “i.e.”

Reply:
Thanks for the suggestion, @chloekraw, but we try to avoid latin abbreviations https://hey.mapbox.com/documentation/style-guide/grammar-and-mechanics/#abbreviations.